README.md

```diff
@@ -72,21 +42,37 @@
+[![Chat](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://chat.vuetifyjs.com)
+[![Version](https://img.shields.io/npm/v/vuetify.svg)](https://www.npmjs.com/package/vuetify)
+[![CDN](https://img.shields.io/cdnjs/v/vuetify.svg)](https://cdnjs.com/libraries/vuetify)
 
-<p>Vuetify is a semantic component framework for Vue. It aims to provide clean, semantic and reusable components that make building your application a breeze.</p>
+## Introduction
 
-<p>Build <i>amazing</i> applications with the power of Vue and Material Design and a massive library of beautifully crafted components. Created according to Google's <strong><a href="https://material.io/" target="_blank">Material Design Spec</a></strong>, Vuetify components feature an easy-to-remember semantic design that shifts remembering complex classes and markup, to type-as-you speak properties that have simple and clear names.</p>
+Vuetify is a semantic component framework for Vue. It aims to provide clean, semantic and reusable components that make building your application a breeze.
 
-<p>Harness the power of the <a href="https://chat.vuetifyjs.com">Vuetify community</a> and get help 24/7 from talented developers across the world, the dev team and the creator, John Leider. Become a <a href="https://www.patreon.com/vuetify">backer</a> and get access to dedicated support from the team.</p>
-  
-<p>Vuetify supports all <strong>modern browsers</strong>, including IE11 and Safari 9+. From mobile to laptop to desktop, you can rest assured that your application will work as expected. Interested in the bleeding edge? Try the vue-cli Webpack SSR (Server side rendered) template and build websites optimized for SEO.</p>
+Build *amazing* applications with the power of Vue and Material Design and a massive library of beautifully crafted components. Created according to Google's **<a href="https://material.io/" target="_blank">Material Design Spec</a>**, Vuetify components feature an easy-to-remember semantic design that shifts remembering complex classes and markup, to type-as-you speak properties that have simple and clear names.
 
-<h2>Demo and Documentation</h2>
+Harness the power of the [Vuetify community](https://chat.vuetifyjs.com) and get help 24/7 from talented developers across the world, the dev team and the creator, John Leider. Become a [backer](https://www.patreon.com/vuetify) and get access to dedicated support from the team.
 
-<a href="https://vuetifyjs.com" target="_blank">Documentation</a>
+### Browser Support
 
-<h2>One minute Quick-start</h2>
+Vuetify supports all **modern browsers**, including IE11 and Safari 9+. From mobile to laptop to desktop, you can rest assured that your application will work as expected. Interested in the bleeding edge? Try the vue-cli Webpack SSR (Server side rendered) template and build websites optimized for SEO.
```

Comment:
Vuetify supports all **modern browsers**, including IE11 and Safari 9+ (using polyfills).